cli/.js/src/main/scala/aqua/remote/DistOpts.scala

```diff
@@ -37,7 +37,7 @@ object DistOpts extends Logging {
 
   def srvNameOpt: Opts[String] =
     Opts
-      .option[String]("service", "Service to deploy from the config file", "s")
+      .option[String]("service", "Service to deploy from the config file", "e")
```

Comment:
Why `-e`? How about `-n` for "service **N**ame"? Or maybe even better would be to avoid using short name here, and use only long? e.g. `--service`